src/app.ts

```diff
@@ -94,13 +95,19 @@ class App {
     }
   }
 
-  private static getOutputFormatter(): OutputFormatter {
-    switch (configs.outputFormat) {
-      case 'default':
-        return defaultFormatter;
-      case 'gitlab':
-        return gitLabFormatter;
+  private static getOutputFileMetas(): OutputFileMeta[] {
+    const outputFileMetas: OutputFileMeta[] = [];
+    if (configs.outputFormat.includes('default')) {
+      outputFileMetas.push({ formatter: defaultFormatter, filename: configs.output });
+    }
+    if (configs.outputFormat.includes('gitlab')) {
+      outputFileMetas.push({
+        formatter: gitLabFormatter,
+        filename: 'gl-code-quality-report.json',
```

Comment:
I think we should let users set the filename for gitlab or any other format if we have in the future as well.